web/sites/default/config/key.key.solr_core.yml

```diff
@@ -9,7 +9,7 @@ key_type: authentication
 key_type_settings: {  }
 key_provider: file
 key_provider_settings:
-  file_location: sites/default/files/private/solr_core.key
```

Comment:
This should not have changed. You may need to cache clear before running a cim. You can revert these changes individually and run cim again.